shell/plugins/dashboard-store/resource-class.js

```diff
@@ -1179,7 +1179,7 @@ export default class Resource {
     }
 
     // @TODO remove this once the API maps steve _type <-> k8s type in both directions
-    opt.data = this.toSave() || { ...this };
+    opt.data = this.toSave() || JSON.parse(JSON.stringify(this)); // Needs to completely dereference self to ensure properties that are cleaned are not removed from cache
```

Comment:
Couldn't this use lodash clone?

Reply:
i stuck with the basic method as it's one less place we have a dependency on lodash. it's a smidge slower but for this use case (run once on user click) it'll be virtually the same